setup.py

```diff
@@ -20,10 +20,30 @@
     description=package['description'] if 'description' in package else package_name,
     long_description=io.open('README.md', encoding='utf-8').read(),
     long_description_content_type='text/markdown',
+    url='https://dash.plot.ly/cytoscape',
+    download_url='https://github.com/plotly/dash-cytoscape/releases',
```

Comment:
Maybe this could be https://pypi.org/project/dash-cytoscape/?

Reply:
Since the `download_url` is accessible from Pypi, it would simply redirect it back to Pypi.org.

Reply:
Ah ok. Well I don't want to commit ourselves to updating the github releases page. We don't do this for any of the other dash repos as it's an extra step involved in publishing. Maybe just leave this out?

Reply:
That's a good point, I'll do that.

Reply:
Done!